### DISCUSSION TITLE
Drag marker on other markers

### DISCUSSION THREAD

**Discussion by vanlueckn** [Leaflet/Leaflet#9395] [Q&A] [answered]
I currently have this code to make a marker draggable:

```
 const marker = L.marker(
      [job.pickUpLocation.latlng.latitude, job.pickUpLocation.latlng.longitude],
      { draggable: true, icon: yellowJobMarker }
    ).addTo(map);
```

How can i check the `dragend` event if the marker was dragged on some other marker?

**Comment by Falke-Design** [comment, marked as answer]
You need to iterate over all other markers and check if the latlng is the same as of the dragged marker
